modules/programs/micro/settings.nix

```diff
@@ -0,0 +1,401 @@
+lib:
+let
+  mkActivatableOption = description:
+    lib.mkOption {
+      default = false;
+      example = true;
+      type = lib.types.bool;
+      inherit description;
+    };
+  mkDeactivatableOption = description:
+    lib.mkOption {
+      default = true;
+      example = false;
+      type = lib.types.bool;
+      inherit description;
+    };
+in {
+  autoindent = mkDeactivatableOption ''
```

Comment:
I would suggest removing all options from this file per [3.2.3. Add only valuable options](https://rycee.gitlab.io/home-manager/index.html#sec-guidelines-valuable-options).